java/src/jmri/jmrix/jmriclient/JMRIClientConfigurationBundle_fr.properties

```diff
@@ -7,7 +7,7 @@
 # These are primarilly used internally, but are being placed
 # here just in case they need to be changed later.
 
-defaultMDNSServerName=localhost
+defaultMDNSServerName=h\u00f4telocal
```

Comment:
I'm not sure if this should be translated - isn't the `localhost` name for referring to the loopback address language non-specific?